docs/css/ez-guidelines-tables.css

```diff
@@ -174,3 +175,7 @@
     font-weight: normal;
     border-radius: 0 0 0.3em 0;
 }
+
+.ez-guidelines-tables__notifications table tr td:first-child:not(.linenos) {
+    width: inherit;
+}
```

Comment:
Nitpick: we used to add empty line at the end of file. 🙂

Reply:
Updated. Thanks @tischsoic